packages/web-comment/src/views/includes/cookie-banner.njk

```diff
@@ -0,0 +1,13 @@
+{% from "../macros/cookie-banner.njk" import cookieBanner %}
+{% from "../macros/cookie-banner-accepted.njk" import cookieBannerAccepted %}
+{% from "../macros/cookie-banner-rejected.njk" import cookieBannerRejected %}
+
+{% set showCookieBanner = displayCookieBanner %}
```

Comment:
don't think displayCookieBanner is defined?

Reply:
Just resolving for now to get the initial merge done then we can pick these points back up